integration-tests/modules/__tests__/cart/store/cart.workflows.spec.ts

```diff
@@ -1280,15 +1269,15 @@ medusaIntegrationTestRunner({
 
           const paymentCollection =
             await paymentModule.createPaymentCollections({
-              amount: 5000,
+              amount: 5001,
```

Comment:
Changed here to test when the value has changed.
if the amount and currency_code are the same, the payment collection isn't updated and the session isn't deleted.